index.html

```diff
@@ -93,10 +93,113 @@ <h3>Extensions to the <code>Crypto</code> interface</h3>
       </p>
       <pre class="idl">
         [Exposed=(Window,Worker)]
-        partial interface <dfn id="dfn-Crypto"></dfn>Crypto</dfn> {
+        partial interface Crypto {
           DOMString randomUUID();
         };
       </pre>
+      <p class="note" role="note"><span>Note:
+        <a href="#random-uuid" data-link-type="dfn">randomUUID()</a></code> returns the
```

Comment:
To be precise, we should also mention that it generates a version 4 UUID:

```suggestion
        <a href="#random-uuid" data-link-type="dfn">randomUUID()</a></code> generates a new
        <a data-cite="RFC4122#section-4.4">version 4 UUID</a> and returns its
        <a data-cite="RFC2141#section-2.2">namespace specific string representation</a> described in
```